requirements/edx/base.in

```diff
@@ -79,7 +79,7 @@ edx-celeryutils
 edx-completion
 edx-django-release-util             # Release utils for the edx release pipeline
 edx-django-sites-extensions
-edx-django-utils>=4.7.0            # Utilities for cache, monitoring, and plugins
+edx-django-utils>=5.0.0            # Utilities for cache, monitoring, and plugins
```

Comment:
Out of curiosity, why are we specifying a min version at all?

Reply:
I'm not totally sure. It's probably because someone needed to upgrade it to a certain version so they could rely on a middleware in edx-platform code, and wanted to make sure that the two changed in sync.

It came in handy today because all I had to do was bump the version there and run `make compile-requirements`, giving me the smallest-necessary requirements change for the upgrade.